src/command_modules/azure-cli-acr/README.rst

```diff
@@ -84,23 +84,36 @@ Update a container registry
 
     Command
         az acr update: Update a container registry.
-
+    
     Arguments
-        --name -n      [Required]: Name of container registry.
-        --disable-admin          : Disable admin user.
-        --enable-admin           : Enable admin user.
-        --resource-group -g      : Name of resource group.
-        --storage-account-name -s: Name of an existing storage account.
```

Comment:
Just sent you an e-mail that should let you simplify this code quite a bit. E-mail me if something isn't clear.